app/views/api/v0/webhooks/index.json.jbuilder

```diff
@@ -1,9 +1,8 @@
 json.array! @webhooks.each do |endpoint|
-  json.type_of "webhook_endpoint"
+  json.type_of        "webhook_endpoint"
   json.id             endpoint.id
+  json.source         endpoint.source
   json.target_url     endpoint.target_url
   json.events         endpoint.events
-  json.source         endpoint.source
   json.created_at     endpoint.created_at.rfc3339
-  json.updated_at     endpoint.updated_at.rfc3339
```

Comment:
I've removed this, because we never expose the `updated_at` for any other resource, because is technically an internal field (it might not correspond to the user's activity on the resource)